editors/vscode/package.json

```diff
@@ -115,7 +115,7 @@
 		},
 		"jsonValidation": [
 			{
-				"fileMatch": "rome.json",
+				"fileMatch": "biome.json",
 				"url": "./configuration_schema.json"
 			}
 		]
```

Comment:
We should **add** one for `biome.json` but we should keep `rome.json`. We still support both